vault/resource_mount.go

```diff
@@ -137,13 +142,23 @@ func mountRead(d *schema.ResourceData, meta interface{}) error {
 
 	log.Printf("[DEBUG] Reading mount %s from Vault", path)
 
-	mount, err := client.Sys().MountConfig(path)
+	mounts, err := client.Sys().ListMounts()
 	if err != nil {
 		return fmt.Errorf("error reading from Vault: %s", err)
 	}
 
-	d.Set("default_lease_ttl_seconds", mount.DefaultLeaseTTL)
-	d.Set("max_lease_ttl_seconds", mount.MaxLeaseTTL)
+	mount, ok := mounts[strings.Trim(path, "/")+"/"]
```

Comment:
So, sometimes `path` has a trailing `/`, but sometimes it doesn't? So here we ensure it has a single `/`? Is that what's happening? Please add some comments here for future readers 😄

Reply:
Updated with comment :D